lib/cell-manager.coffee

```diff
@@ -1,57 +1,27 @@
-_ = require 'lodash'
-
 module.exports = CellManager =
-    _setBreakpointDecorationAt: (editor, row) ->
-        marker = editor.markBufferPosition
-            row: row
-            column: 0
-        ,
-            invalidate: 'never'
-
-        editor.decorateMarker marker,
-            type: 'line-number'
-            class: 'breakpoint'
-
-    _getBreakprointDecorationAt: (editor, row) ->
-        decorations = editor
-            .getLineNumberDecorations({class: 'breakpoint'})
-            .filter (decoration) ->
-                return decoration.marker.getStartBufferPosition().row is row
-        return decorations[0]
-
-    removeAllBreakpoints: ->
-        editor = atom.workspace.getActiveTextEditor()
-        decorations = editor.getLineNumberDecorations({class: 'breakpoint'})
-        for decoration in decorations
-            decoration.marker.destroy()
-
-    toggleBreakpoint: ->
-        editor = atom.workspace.getActiveTextEditor()
-        row = editor.getLastCursor().getBufferRow()
-        decoration = CellManager._getBreakprointDecorationAt editor, row
-
-        if decoration
-            decoration.marker.destroy()
-            return
-
-        CellManager._setBreakpointDecorationAt editor, row
-
     getCurrentCell: ->
         editor = atom.workspace.getActiveTextEditor()
+        buffer = editor.getBuffer()
         row = editor.getLastCursor().getBufferRow()
-        decorations = editor.getLineNumberDecorations({class: 'breakpoint'})
+        scope = editor.scopeDescriptorForBufferPosition([row, 0])
```

Comment:
I think [editor.getRootScopeDescriptor](https://atom.io/docs/api/v1.8.0/TextEditor#instance-getRootScopeDescriptor) is better here. Otherwise things like the below would be possible:

```coffeescript
`
//%%
(function() {console.log("This is Javascript");})();
//%%
`
console.log 'This is coffeescript'
```